plugins/nid-team/scripts/pr-dashboard/list-merged-without-reviewer.py

```
@@ -0,0 +1,56 @@
+
+import json
+import subprocess
+import datetime
+import sys
+
+days = int(sys.argv[1]) if len(sys.argv) > 1 else 7
+cutoff = datetime.datetime.now(datetime.timezone.utc) - datetime.timedelta(days=days)
+
+raw = subprocess.run(
+    ["gh", "project", "item-list", "28", "--owner", "openshift",
+     "--format", "json", "--limit", "500"],
```

Ruff (0.16.2)

[error] Starting a process with a partial executable path

(S607)